azureml/pipelines/submit_knn_fewshot.py

```diff
@@ -0,0 +1,180 @@
+import time
+
+from dataclasses import dataclass
+
+import hydra
+from hydra.core.config_store import ConfigStore
+
+import omegaconf
+
+from azure.identity import DefaultAzureCredential
+from azure.ai.ml import MLClient
+
+from azure.ai.ml import dsl, MLClient, Input
+from azure.ai.ml.entities import Pipeline
+
+from azureml_utils import get_component_collector, ComponentCollector
+from configs import AMLConfig, KNNFewshotConfig, AOAIConfig
+from constants import GUIDANCE_PROGRAMS_DIR
+from logging_utils import get_standard_logger_for_file
+
+_logger = get_standard_logger_for_file(__file__)
+
+
+@dataclass
+class PipelineConfig:
+    knn_fewshot_config: KNNFewshotConfig = omegaconf.MISSING
+    azureml_config: AMLConfig = omegaconf.MISSING
+    aoai_config: AOAIConfig = omegaconf.MISSING
+    aoai_embedding_config: AOAIConfig = omegaconf.MISSING
+
+
+cs = ConfigStore.instance()
+cs.store(name="config", node=PipelineConfig)
+
+
+def create_embedding_for_split_pipeline(
```

Comment:
Long term, this isn't the subpipeline we'd want, but this serves for getting-things-started